index.js

```diff
@@ -5,17 +5,30 @@ const BinWrapper = require('bin-wrapper');
 
 const version = '0.33.3'
 const base = 'https://github.com/saucelabs/saucectl/releases/download';
-const bin = new BinWrapper()
-	.src(`${base}/v${version}/saucectl_${version}_mac_32-bit.tar.gz`, 'darwin', 'x86')
-	.src(`${base}/v${version}/saucectl_${version}_mac_64-bit.tar.gz`, 'darwin', 'x64')
-	.src(`${base}/v${version}/saucectl_${version}_linux_32-bit.tar.gz`, 'linux', 'x86')
-	.src(`${base}/v${version}/saucectl_${version}_linux_64-bit.tar.gz`, 'linux', 'x64')
-	.src(`${base}/v${version}/saucectl_${version}_win_32-bit.zip`, 'win32', 'x86')
-	.src(`${base}/v${version}/saucectl_${version}_win_32-bit.zip`, 'win32', 'x64')
-	.src(`${base}/v${version}/saucectl_${version}_win_64-bit.zip`, 'win64', 'x64')
-    .dest(path.join(__dirname, 'bin'))
-    .use(process.platform.startsWith('win') ? 'saucectl.exe' : 'saucectl')
-	.version(`v${version}`);
+const binWrapper = (binInstallURL) => {
+    const bw = new BinWrapper();
+
+    if (binInstallURL) {
+        bw.src(binInstallURL, '', '')
+            .dest(path.join(__dirname, 'bin'))
+            .use(process.platform.startsWith('win') ? 'saucectl.exe' : 'saucectl');
+        return bw;
+    }
+
+    bw.src(`${base}/v${version}/saucectl_${version}_mac_32-bit.tar.gz`, 'darwin', 'x86')
+        .src(`${base}/v${version}/saucectl_${version}_mac_64-bit.tar.gz`, 'darwin', 'x64')
+        .src(`${base}/v${version}/saucectl_${version}_mac_64-bit.tar.gz`, '', '')
```

Comment:
@valentyn88 why does this one has an empty platform and arch?

Reply:
I forgot to delete this line. Now everything is ok.